package/MDAnalysis/analysis/gnm.py

```diff
@@ -337,12 +353,9 @@ def generate_kirchoff(self):
             for jcounter in neighbour_atoms:
                 if jcounter > icounter and _dsq(positions[icounter], positions[jcounter]) <= cutoffsq:
                     iresidue, jresidue = residue_index_map[icounter], residue_index_map[jcounter]
-                    if self.MassWeight:
-                        contact = 1.0 / (len(self.ca.residues[iresidue].atoms) * len(self.ca.residues[jresidue].atoms)) ** 0.5
```

Comment:
Will fix elsewhere; apparently only mattered in the instance were I fixed it because many more atoms were used.

I can try to disentangle but their were also some cleanups in gnm in an earlier commit. I will look at a local interactive rebase and see if I can merge these commits.